metadata/lib.rs

```diff
@@ -0,0 +1,629 @@
+#![deny(missing_docs)]
```

Comment:
Can this just be `warn` and rely on the `-Dwarnings` in CI to avoid blocking developers locally.

Reply:
Good idea, done.